test/functorch/test_aotdispatch.py

```diff
@@ -6010,6 +6010,18 @@ def forward(self, x):
         with self.assertRaisesRegex(AssertionError, "Unexpected fake"):
             aot_module_simplified(MockModule(), (fake_x,), nop)
 
+    def test_aot_test_subclasses_with_tensor_factories(self):
+        from torch.testing._internal.test_subclasses import SubclassWithTensorFactory
+
+        inp = SubclassWithTensorFactory(torch.zeros(3, 5))
+
+        def fn(x):
+            return 2 * x
+
+        ref_out = fn(inp)
+        out = torch.compile(fn, backend="aot_eager", fullgraph=True)(inp)
+        # self.assertEqual(ref_out, out)
```

Comment:
what's up here?

Reply:
Yeah, I commented it, forgot to remove. 
Assert fails on some op that is not implemented in my Testing Subclass.

Reply:
Resolved, changed impl fn to * ones only for float32